src/common/queries/expenses.ts

```diff
@@ -90,23 +90,36 @@ export function useExpensesQuery(params: ExpensesParams) {
   );
 }
 
+const successMessages = {
+  bulk_update: 'updated_records',
+};
+
 export function useBulk() {
   const queryClient = useQueryClient();
   const invalidateQueryValue = useAtomValue(invalidationQueryAtom);
 
-  return (
+  return async (
```

Comment:
Am I missing something or is there a reason these fns are marked as async? 
No awaits inside?